terraform/meta/main.tf

```diff
@@ -43,4 +43,7 @@ module "prod_backend" {
 
 module "ecr" {
   source = "../modules/ecr"
+
+  # This will need updating to include dev and production roles
+  allow_access_by_roles = ["arn:aws:iam::107155005276:role/core-stag-task-execution"]

```

Comment:
Hardcoding this is obviously annoying (was alright with the account ids, but this seems much more painful).

It'll do for now - will need to think through ordering and what dependencies we're happy to have where in order to do a nicer set up where these arns get picked up automatically, which we should do but can be at a later point.

Reply:
https://dluhcdigital.atlassian.net/browse/CLDC-2675